<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#131239;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>420</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Currently this level is not unlocked !</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="f_ok_level">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial Rounded MT Bold&quot;;
padding:5px;
background-color:green;
color:white;

</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
